services/navet-ms/serverless.yml

```diff
@@ -17,7 +16,6 @@ provider:
   region: eu-north-1
   endpointType: regional
   tracing:
-    apiGateway: true
```

Comment:
I'm not sure what `tracing` does but isn't it useful for debugging? Why is `apiGateway` irrelevant? Is it because `navet-ms` is never triggered by API requests, only `bankid.collect`? Does it affect CloudWatch/logging?

Reply:
You are right @KenthLjungHBG the navet-ms is not related to the apiGateway. Do not think this will change anything at all for logging purposes, but not 100% sure

Reply:
True @TeddyAndersson